dockerfile/api/Dockerfile

```diff
@@ -1,5 +1,6 @@
-ARG AWS_REGION
-FROM 243963068353.dkr.ecr.${AWS_REGION}.amazonaws.com/builder:latest as build
+ARG DOCKERHUB_URL
+FROM ${DOCKERHUB_URL}builder:latest as build
```

Comment:
I assume this url will still be the same for the staging and prod environment right?

Reply:
Let me know whenever you update this path, especially adding a new ARG since I need to change my script to specify the ARG